src/guide/builtin-helpers.md

```diff
@@ -27,6 +27,17 @@ section, marked by `else` is called an "else section".
 
 <ExamplePart examplePage="/examples/builtin-helper-ifelse-block.md" show="template" />
 
+### includeZero
+
+The `includeZero=true` option may be set to treat the conditional as not empty.
+This effectively determines if `0` is handled by the positive or negative path.
+
+```handlebars
+{{#if 0 includeZero=true}}
+<h1>Does render</h1>
+{{/if}}
+```
```

Comment:
Proof that this example works: 

https://handlebarsjs.com/playground.html#format=1&currentExample=%7B%22template%22%3A%22%7B%7B%23if%200%20includeZero%3Dtrue%7D%7D%5Cn%3Ch1%3EDoes%20render%3C%2Fh1%3E%5Cn%7B%7B%2Fif%7D%7D%22%2C%22partials%22%3A%5B%5D%2C%22input%22%3A%22%7B%7D%5Cn%22%2C%22output%22%3A%22%3Ch1%3EDoes%20render%3C%2Fh1%3E%5Cn%22%2C%22preparationScript%22%3A%22%22%2C%22handlebarsVersion%22%3A%224.7.7%22%7D